### PR TITLE
BH-1216: ACM Documentation 0.6

### PR DESCRIPTION
**Pull request by matthewguywright**

### CODE REVIEW

File: content-engine/tutorial/models/delete.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ![Content Engine sync](../../images/delete-model.png)
+
+1. Confirm deletion by clicking Delete in the modal that pops up. Click Cancel to keep the model
+    
+    ![Content Engine sync](../../images/Screen_Shot_2021-09-01_at_12.08.32_PM.png)
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
+- **Feedback**: Your feedback helps shape the future of Content Engine. To provide feedback, fill out our [Content Engine Feedback Form](https://docs.google.com/forms/d/e/1FAIpQLSecvuZ_EMiTIOlTSwcW1JnPQcFbAcCOwGlhURkzBI8Ps9vFzA/viewform).
```
Comment (anthonyburchell) [inline content-engine/tutorial/models/delete.mdx:32]: Do we have one of these forms for ACM? When I follow the link it takes me to one that says "Content Engine Feedback"

Also same for the Questions section `ce-beta@wpengine.com` might need to be acm-beta@wpengine if we can get the distro made.

Comment (anthonyburchell) [reply content-engine/tutorial/models/delete.mdx:32]: NEVERMIND. this is content engine.

**Comment by anthonyburchell** [comment]
Looks like .DS_Store is added to this PR. We may want to remove that one.

**Comment by wjohnsto** [comment]
Yeah add a .gitignore and make sure at least `.DS_Store`  and `.idea` is in it (then resync).

Also, these appear to be ACM docs, but they are under `content-engine`. Do we want to do it this way or do we want a `content-modeler` root folder with a new config?

**Comment by mindctrl** [comment]
> Do we want to do it this way or do we want a content-modeler root folder with a new config?

I think we want the new folder.

**Comment by wjohnsto** [comment]
> 
> 
> > Do we want to do it this way or do we want a content-modeler root folder with a new config?
> 
> I think we want the new folder.

Agreed

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 .vscode
-.code-workspace
\ No newline at end of file
+.code-workspace
+content-engine/.DS_Store
```
Comment (ChrisWiegman) [inline .gitignore:3]: You should ignore .DS_Store everywhere, not just in a subfolder.

**Review by ChrisWiegman — CHANGES_REQUESTED** [review]
The documentation seems OK... One minr logistical change could help developers later.

File: .gitignore (hunk 2 above)
Comment (wjohnsto) [reply .gitignore:3]: Also ignore `.idea` and [resync the repo with the .gitignore](https://stackoverflow.com/questions/7075923/resync-git-repo-with-new-gitignore-file)

File: atlas-content-modeler/tutorial/graphql/introduction.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+slug: /content-engine/tutorial/graphql/introduction
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/graphql/introduction.mdx:2]: should be `/atlas-content-modeler/tutorial/graphql/introduction`

File: atlas-content-modeler/tutorial/models/create.mdx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+slug: /content-engine/tutorial/models/create
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/create.mdx:2]: should be `/atlas-content-modeler/tutorial/models/create`

File: atlas-content-modeler/tutorial/models/delete.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+slug: /content-engine/tutorial/models/delete
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/delete.mdx:2]: should be `/atlas-content-modeler/tutorial/models/delete`

File: atlas-content-modeler/tutorial/models/create.mdx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+title: Content Engine Models - Create
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/create.mdx:3]: Maybe a different title: `Content Modeler Models - Create`

File: atlas-content-modeler/tutorial/graphql/introduction.mdx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+title: Content Engine - GraphiQL
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/graphql/introduction.mdx:3]: Maybe a different title: `Content Modeler - GraphiQL`

File: atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+    ![Content Engine sync](../../../../images/Screen_Shot_2021-09-01_at_12.18.18_PM.png)
+
+1. Click Create when done to save the field to the model
+
+1. Click Cancel to stop field creation
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx:31]: Is `ce-beta@wpengine.com` the correct support email for Content Modeler?

File: atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+- **Feedback**: Your feedback helps shape the future of Content Engine. To provide feedback, fill out our [Content Engine Feedback Form](https://docs.google.com/forms/d/e/1FAIpQLSecvuZ_EMiTIOlTSwcW1JnPQcFbAcCOwGlhURkzBI8Ps9vFzA/viewform).
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx:32]: This is a Content Engine feedback form, is that right for Content Modeler?

File: atlas-content-modeler/tutorial/models/fields/richtext/introduction.mdx [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+    ![Content Engine sync](../../../../images/Screen_Shot_2021-09-01_at_12.04.46_PM.png)
+
+1. Click Create when done to save the field to the model
+
+1. Click Cancel to stop field creation
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/fields/richtext/introduction.mdx:31]: Is ce-beta@wpengine.com the correct support email for Content Modeler?

File: atlas-content-modeler/tutorial/models/fields/richtext/introduction.mdx [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+- **Feedback**: Your feedback helps shape the future of Content Engine. To provide feedback, fill out our [Content Engine Feedback Form](https://docs.google.com/forms/d/e/1FAIpQLSecvuZ_EMiTIOlTSwcW1JnPQcFbAcCOwGlhURkzBI8Ps9vFzA/viewform).
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/fields/richtext/introduction.mdx:32]: This is a Content Engine feedback form, is that right for Content Modeler?

File: atlas-content-modeler/tutorial/models/fields/text/index.mdx [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+1. Click on Advanced Settings to choose a minimum and maximum character length
+
+1. Click Create when done to save the field to the model
+
+1. Click Cancel to stop field creation
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
+- **Feedback**: Your feedback helps shape the future of Content Engine. To provide feedback, fill out our [Content Engine Feedback Form](https://docs.google.com/forms/d/e/1FAIpQLSecvuZ_EMiTIOlTSwcW1JnPQcFbAcCOwGlhURkzBI8Ps9vFzA/viewform).
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/fields/text/index.mdx:38]: This is a Content Engine feedback form, is that right for Content Modeler?

File: atlas-content-modeler/tutorial/models/fields/text/index.mdx [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Click on Advanced Settings to choose a minimum and maximum character length
+
+1. Click Create when done to save the field to the model
+
+1. Click Cancel to stop field creation
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
```
Comment (wjohnsto) [inline atlas-content-modeler/tutorial/models/fields/text/index.mdx:37]: Is ce-beta@wpengine.com the correct support email for Content Modeler?

**Review by wjohnsto — CHANGES_REQUESTED** [review]
I started to mark them all, but there are many references of "Content Engine" that should be "Content Modeler". There are also references to the Content Engine email (i.e. `ce-beta@wpengine.com`) and Content Engine Feedback Form.

Content Modeler is not Content Engine. I could be wrong, but I don't think any of the references to Content Engine in the Content Modeler docs are appropriate.

File: atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx (hunk 8 above)
Comment (mindctrl) [reply atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx:31]: I don't think so. Looks like a CE thing. We don't have an official feedback/support email. We're planning to handle that through GitHub and wp.org forums.

File: atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx (hunk 9 above)
Comment (mindctrl) [reply atlas-content-modeler/tutorial/models/fields/relationship/introduction.mdx:32]: Link should be: https://docs.google.com/forms/d/e/1FAIpQLScc2VN-GRSJMz8zVgJLL6kiX3VeV2jkSDnmU1gnuNElEHCEVQ/viewform?usp=sf_link

We could also link to GitHub issues/discussion, and wp.org support forums in the future (we don't have that URL secured yet).

**Comment by mindctrl** [comment]
@matthewguywright are you still planning to rename the images so the filename describes the screenshot?

File: atlas-content-modeler/tutorial/models/fields/text/index.mdx (hunk 12 above)
Comment (mindctrl) [reply atlas-content-modeler/tutorial/models/fields/text/index.mdx:38]: Should be same link as provided above.  https://github.com/wpengine/headless-docs/pull/71#discussion_r706344489

File: atlas-content-modeler/tutorial/models/fields/text/index.mdx (hunk 13 above)
Comment (mindctrl) [reply atlas-content-modeler/tutorial/models/fields/text/index.mdx:37]: @matthewguywright same as https://github.com/wpengine/headless-docs/pull/71#discussion_r706343782

**Pull request closed without merging**